pkg/kubelet/util.go

```diff
@@ -50,7 +49,7 @@ func InitHealthChecking(k *Kubelet) {
 	// TODO: These should probably become more plugin-ish: register a factory func
 	// in each checker's init(), iterate those here.
 	health.AddHealthChecker(health.NewExecHealthChecker(k))
-	health.AddHealthChecker(health.NewHTTPHealthChecker(&http.Client{}))
+	health.AddHealthChecker(health.NewHTTPHealthChecker(health.Timeout))
```

Comment:
Is there a preferred way of doing this by a utils function (it wasn't obvious from a skim of Wait)? If not, I've set up the typical select over time.After and a channel returning the regular Exec results.